src/lib/Behat/PageObject/RolePage.php

```diff
@@ -104,4 +104,37 @@ public function startAssigningToItem(string $roleName): void
         $this->navLinkTabs->goToTab('Assignments');
         $this->adminLists['Assignments']->clickAssignButton();
     }
+
+    /**
+     * Verifies if Role with Limitation from given list is present.
+     *
+     * @param string $listName
+     * @param string $moduleAndFunction
+     * @param string $limitation
+     *
+     * @return bool
+     */
+    public function isRoleWithLimitationPresent(string $listName, string $moduleAndFunction, string $limitation): bool
+    {
+        $policyExists = false;
+
+        $this->navLinkTabs->goToTab($listName);
+        $adminList = $this->adminLists[$listName];
+        $actualPoliciesList = $adminList->table->getTableHash();
+
+        $expectedModule = explode('/', $moduleAndFunction)[0];
+        $expectedFunction = explode('/', $moduleAndFunction)[1];
```

Comment:
This could be simplified to: 

```php
list($expectedModule, $expectedFunction) = explode('/', $moduleAndFunction);
```

but both versions make an assumption that '/' is always present in the `$moduleAndFunction` string 😉

Reply:
I'm against `list()` - explicit variable declaration is more readable and it easier to understand for ~stupid~ less intelligent IDEs.